styleguide.config.js

```diff
@@ -50,6 +50,15 @@ module.exports = {
     {
       name: 'Templates',
       components: 'src/templates/**/*.js'
+    },
+    {
+      name: 'Pages',
```

Comment:
@cisacke I thought about it, but we need an example for the navigator... anyway and I didn't want to just make a dummy example or have two of the same. What do you think?